sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -45,7 +46,7 @@ public BlobTestBase(bool async, RecordedTestMode? mode = null)
 
         public BlobClientOptions GetOptions(bool parallelRange = false)
         {
-            var options = new BlobClientOptions
+            var options = new BlobClientOptions((BlobClientOptions.ServiceVersion) _serviceVersion)
```

Comment:
Nit - we don't need the cast, do we?

Reply:
Not anymore, leftovers from previous iteration.